Gemfile

```diff
@@ -1,3 +1,5 @@
+ruby '2.7.6'
```

Comment:
> By default, the Gemfile utilizes system Ruby, which unfortunately does not support ARM Macs well.

Curious what you mean by this. I'd think/hope the system Ruby shipped on ARM Macs worked OK but maybe that's not the case?

Reply:
What I know: running `bundle exec pod trunk push` fails with Apple's ruby. But with later versions it works.

With Apple's ruby, we fail loading `ffi`, and the stated solution is to bump the ruby version: see https://github.com/ffi/ffi/issues/922

Reply:
Maybe macOS 13 resolves this issue? But for now this seems reasonable

Reply:
Yep seems very reasonable to enforce a Ruby version 👍